lib/iris/fileformats/rules.py

```diff
@@ -735,7 +735,8 @@ def _ensure_aligned(regrid_cache, src_cube, target_cube):
 
 Loader = collections.namedtuple('Loader',
                                 ('field_generator', 'field_generator_kwargs',
-                                 'converter', 'legacy_custom_rules'))
+                                 'converter', 'legacy_custom_rules',
+                                 'fault_tolerant'))
```

Comment:
As it stands, this is a backwards incompatible change. How much of an issue this is depends on whether it's targeting 1.9 or 2.0. (This kind of thing is why we might want to make more bits of the Iris API private.)

If we want to keep backwards compatibility then I think we need something like the following:
```python
class Loader(collections.namedtuple(...)):
    __slots__ = ()
    def __new__(cls, field_generator, ..., fault_tolerant=False):
        return tuple.__new__(cls, field_generator, ..., fault_tolerant)
```